example/ios/PagerViewExample/AppDelegate.mm

```diff
@@ -23,4 +23,14 @@ - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
 #endif
 }
 
+/// This method controls whether the `concurrentRoot`feature of React18 is turned on or off.
```

Comment:
Do we need that?

Reply:
Related to https://github.com/callstack/react-native-pager-view/pull/783#discussion_r1408954881, let's keep the discussion there.